plugins/lime-plugin-pirania/src/pages/create.js

```diff
@@ -14,12 +14,17 @@ import Loading from '../../../../src/components/loading';
 import daysFromNow from '../../../../src/utils/daysFromNow';
 import makeid from '../../../../src/utils/makeid';
 
-function Create({ goBack, createMemberVoucher, createVisitorVoucher, daysLeft, createVoucher, loading }) {
+function Create({ goBack, createMemberVoucher, createVisitorVoucher, daysLeft, createVoucher, loading, date }) {
```

Comment:
This function feels it "knows too much", having a lot of parameters sometimes says that is describing a structure that is not being created... these values could all be together an object that has sense in itself... not an urgent change, but something to reflect on